auth0/src/test/java/com/auth0/android/request/internal/BaseRequestTest.kt

```diff
@@ -275,30 +275,32 @@ public class BaseRequestTest {
     public fun shouldExecuteRequestOnBackgroundThreadAndPostSuccessToMainThread() {
         val pausedExecutorService = PausedExecutorService()
         val threadSwitcher =
```

Comment:
nit: we could avoid most of the diff in this file if we rename `threadSwitcher` to `defaultThreadSwitcher` and later call the spy `threadSwitcher`.

Reply:
That's a good call out. Will update accordingly